packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -327,6 +327,7 @@ const CIDocument: FC<CIDocumentProps> = ({
             selected={nonContractTabs.indexOf(selectedType)}
             onSelectionChange={(index: number): void => setSelectedType(nonContractTabs[index])}
             aria-label={messages.filtersTabLabel}
+            data-testid="tabs"
```

Comment:
I think I'd prefer most descriptive / unique `testid` names. That being said, I'm bad at naming things. 😆 Maybe `noncontract-tabs` or `cidocument-noncontract-tabs`?